src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// import classNames from 'classnames';
+
+import { TodoInfo } from '../TodoInfo';
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/Todo';
+
+import './TodoList.scss';
+
+type Todos = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Todos> = ({ todos }) => (
+  <ul className="TodoList">
+    {todos.map((todoItem: Todo) => (
+      <li className={(todoItem.completed)
```

Comment:
You should add unique key for each li element
```suggestion
      <li key={...} className={(todoItem.completed)
```